### ISSUE TITLE
method POST not working properly in KrakenD

### ISSUE THREAD

**Issue by Svastikkka** [krakendio/krakend-ce#508]
**Environment info:**

* KrakenD version: Version: 2.0.0-ee
* System info: docker
* Backend technology: Java
* Additional environment information: I am using AWS eks fargate to enable KrakenD api gateway for my microservices.

**Describe what are you trying to do**:
Hi, I have a webhook consumer microservice running in the Kubernetes cluster.  Webhook consumer only allows POST requests so I try to send a POST request through krakenD but it is not working properly and giving me 500 Internal Server Error in response. The response I wanted is  200

Using webhook consumer microservice
URL: webhook-service-name:29191/webhooks/alm/rally/token  -> Gives me 200 response

Using krakenD  
URL: localhost:8080/webhooks/alm/rally/token  -> Gives me 500 response

Notes 
- I am using POSTMAN for sending request.
-  Without krakenD if i send request direct to my webhook consumer microservice it works fine and give me 200 response.
- I already check my microservice k8s service name and it is correct.

**Your configuration file**:

```json
    {
      "$schema": "https://www.krakend.io/schema/v3.json",
      "version": 3,
      "name": "KrakenD - API Gateway",
      "extra_config": {
        "telemetry/opencensus": {
          "sample_rate": 100,
          "reporting_period": 1,
          "exporters": {
            "prometheus": {
              "port": 9091
            }
          }
        },
        "plugin/http-server": {
          "name": [
            "wildcard"
          ],
          "wildcard": {
            "endpoints": {
              "/webhooks": [
                "/webhooks"
              ]
            }
          }
        }
      },
      "timeout": "3000ms",
      "cache_ttl": "300s",
      "output_encoding": "string",
      "disable_rest": true,
      "sequential_start": true,
      "endpoints": [
        {
          "endpoint": "/webhooks",
          "method": "POST",
          "output_encoding": "json",
          "backend": [
            {
              "url_pattern": "/webhooks/alm/rally",
              "encoding": "string",
              "sd": "static",
              "method": "POST",
              "host": [ "http://webhooks-svc:29191" ],
              "extra_config": {
                "plugin/http-client": {
                  "name": "wildcard"
                }
              }
            }
          ],
          "input_headers": [
            "X-Krakend-Wildcard"
          ]
        }
      ],
      "plugin": {
        "pattern": ".so",
        "folder": "/opt/krakend/plugins/"
      }
    }
```

**Configuration check output**:
Result of `krakend check -dtc krakend.json --lint` command

```
Global settings
        Name: KrakenD - API Gateway
        Port: 8080
        Folder: /opt/krakend/plugins/
        Pattern: .so
2 global component configuration(s):
- plugin/http-server
- telemetry/opencensus
1 API endpoint(s):
        - POST /webhooks
        Timeout: 3s
        Connecting to 1 backend(s):
                [+] POST /webhooks/alm/rally
                Timeout: 3s
                Hosts: []
                1 backend component configuration(s):
                - plugin/http-client

0 async agent(s):
Syntax OK!
```

**Commands used:**
How did you start the software?
```
K8S YAML
command: [ "/usr/bin/krakend" ]
args: [ "run", "-d", "-c", "/etc/krakend/krakend.json","--accept-eula", "-p", "8080" ]
```

**Logs:**
```
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Modifier Plugin] plugin #8 (/opt/krakend/plugins/no-redirect.so): plugin: symbol ModifierRegisterer not found in plugin gitlab.com/devops-faith/krakend-ee/v2/plugin/http/client/redirect
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Modifier Plugin] plugin #9 (/opt/krakend/plugins/redis-ratelimit.so): plugin: symbol ModifierRegisterer not found in plugin gitlab.com/devops-faith/krakend-ee/v2/plugin/http/handler/redis-ratelimit
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Modifier Plugin] plugin #10 (/opt/krakend/plugins/url-rewrite.so): plugin: symbol ModifierRegisterer not found in plugin gitlab.com/devops-faith/krakend-ee/v2/plugin/http/handler/url-rewrite
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Modifier Plugin] plugin #11 (/opt/krakend/plugins/virtualhost.so): plugin: symbol ModifierRegisterer not found in plugin gitlab.com/devops-faith/krakend-ee/v2/plugin/http/handler/virtualhost
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Modifier Plugin] plugin #12 (/opt/krakend/plugins/wildcard.so): plugin: symbol ModifierRegisterer not found in plugin gitlab.com/devops-faith/krakend-ee/v2/plugin/http/handler/wildcard
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Plugin Loader] Loading process completed
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: OpenCensus] Service correctly registered
2022/06/13 17:53:14 KRAKEND DEBUG: [SERVICE: Gin] Debug enabled
2022/06/13 17:53:14 KRAKEND INFO: Starting the service
2022/06/13 17:53:14 KRAKEND DEBUG: [ENDPOINT: /webhooks] Building the proxy pipe
2022/06/13 17:53:14 KRAKEND DEBUG: [BACKEND: /webhooks/alm/rally] Injecting plugin wildcard
2022/06/13 17:53:14 KRAKEND DEBUG: [ENDPOINT: /webhooks][JWTSigner] Signer disabled
2022/06/13 17:53:14 KRAKEND INFO: [ENDPOINT: /webhooks][JWTValidator] Validator disabled for this endpoint
2022/06/13 17:53:14 KRAKEND INFO: [SERVICE: Gin] Listening on port: 8080
[PLUGIN: wildcard]: Handler plugin enabled2022/06/13 17:53:14 KRAKEND DEBUG: [PLUGIN: Server] Injecting plugin wildcard
 2022/06/13 - 17:53:59.512 [AccessLog] | 405 |    84.02µs |       127.0.0.1 | GET      /webhooks
 2022/06/13 - 17:54:14.048 [AccessLog] | 500 |  120.976µs |       127.0.0.1 | POST     /webhooks
 2022/06/13 - 17:54:16.049 [AccessLog] | 500 |   80.113µs |       127.0.0.1 | POST     /webhooks
 2022/06/13 - 17:54:17.226 [AccessLog] | 500 |   80.946µs |       127.0.0.1 | POST     /webhooks
 2022/06/13 - 17:54:24.204 [AccessLog] | 500 |   83.561µs |       127.0.0.1 | POST     /webhooks
 2022/06/13 - 17:54:29.214 [AccessLog] | 500 |   72.372µs |       127.0.0.1 | POST     /webhooks

```

**Additional comments:**
Please help

**Comment by Svastikkka** [comment]
Update 
From my microservice i am receiving UNKNOWN content-type

```
2022-06-14 05:08:13.882  WARN 1 --- [io-29191-exec-2] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type '' not supported]
```

**Comment by kpacha** [comment]
check this documentation link: https://www.krakend.io/docs/endpoints/parameter-forwarding/

TLDR; by default, krakend ignores all the headers from the original request. Add the ones you backend is expecting at the `input_headers` array
